codegenerator/cli/npm/envio/src/db/EntityHistory.res

```diff
@@ -144,7 +144,7 @@ let insertRow = (
   ~historyRow: historyRow<'entity>,
   ~shouldCopyCurrentEntity,
 ) => {
-  let row = historyRow->S.serializeOrRaiseWith(self.schema)
+  let row = historyRow->S.reverseConvertToJsonOrThrow(self.schema)
```

Comment:
What is reverseConvertToJsonOrThrow again?

Reply:
This is the same as serializeOrRaiseWith but with a different name. I've written about the change here https://dev.to/dzakh/rescript-schema-v9-zod-like-library-to-the-next-level-1dn6

Reply:
Ok cool, I'm still trying to wrap my head around what is the convert step and why is it reversed

Reply:
It's convert because there's no type validation - otherwise it'd be called parse
It's reversed since we're turning 'value to Js.Json.t instead of Js.Json.t to 'value